core_contracts/governance/governance.py

```diff
@@ -823,12 +823,21 @@ def balanceSetUnstakingPeriod(self, _time: int) -> None:
         baln = self.create_interface_score(self.addresses['baln'], BalancedInterface)
         baln.setUnstakingPeriod(_time)
 
-    @external
-    @only_owner
-    def daoDisburse(self, _recipient: Address, _amounts: List[Disbursement]) -> None:
+    def daoDisburse(self, _recipient: str, _amounts: List[Disbursement]):
+        if len(_amounts) > 3:
+            revert(f"Cannot disburse more than 3 assets at a time.")
+        _recipient = Address.from_string(_recipient)
+
+        for disbursement in _amounts:
+            disbursement['address'] = Address.from_string(disbursement['address'])
         dao = self.create_interface_score(self.addresses['daofund'], DAOfundInterface)
         dao.disburse(_recipient, _amounts)
 
+    @external
+    @only_owner
+    def daoDistribute(self, _recipient: str, _amounts: List[Disbursement]) -> None:
```

Comment:
Do we need this? I forget if there was a reason to have it. Seems like disbursements should only happen as a result of a vote through governance.

Reply:
No , there was this method with only owner decorator before. so we decided to leave it as it is and create another internal method to call vote action. And this method can be called by owner later if required. If that is not required we can remove that.

Reply:
So that means every time  we want to call daoDisburse we need to go through voting first ? if this is the case we don't need that external method.